pkg/server/container_create_unix.go

```diff
@@ -168,10 +168,7 @@ func (c *criService) containerSpec(id string, sandboxID string, sandboxPid uint3
 		if !sandboxConfig.GetLinux().GetSecurityContext().GetPrivileged() {
 			return nil, errors.New("no privileged container allowed in sandbox")
 		}
-		specOpts = append(specOpts, oci.WithPrivileged)
-		if !ociRuntime.PrivilegedWithoutHostDevices {
-			specOpts = append(specOpts, customopts.WithPrivilegedDevices)
```

Comment:
I suspect you should just replace WithPrivilegedDevices with WithHostDevices. @Random-Liu thoughts?

Reply:
Yeah, and remove the code for `WithPrivilegedDevices`.